governance/votes/Executive vote - November 29, 2023.md

```diff
@@ -0,0 +1,146 @@
+---
+title: Template - [Executive Vote] PSM-GUSD-A Removal from Cash Stablecoins, Coinbase Custody Debt Ceiling Increase, Various Parameter Changes, RWA Foundation Dao Resolution, Andromeda Legal Expenses, Vest Streams Updates and Spark Proxy-Spell - November 29, 2023
+summary: Setting PSM-GUSD-A Debt Ceiling to zero and removing from Autoline, Coinbase Custody (RWA014-A) Debt Ceiling increase, WBTC-A, WBTC-B, and WBTC-C Stability Fee changes, SBE parameter changes, RWA Foundation Dao Resolution, Andromeda Legal Expenses, Cease Vest Streams for Steakhouse, Set up Ecosystem Facilitator Vest Streams and TechOps and Spark proxy-spell
```

Comment:
See comment lower down. The contract is call AutoLine

The Techops streams are being `yank`ed. Not created.

```suggestion
summary: Setting PSM-GUSD-A Debt Ceiling to zero and removing from AutoLine, Coinbase Custody (RWA014-A) Debt Ceiling increase, WBTC-A, WBTC-B, and WBTC-C Stability Fee changes, SBE parameter changes, RWA Foundation Dao Resolution, Andromeda Legal Expenses, Cancel Vesting Streams for Steakhouse and TechOps, Set up Ecosystem Facilitator Vesting Streams and Spark proxy-spell
```